### DISCUSSION TITLE
Remove obsolete check

### DISCUSSION THREAD

**Discussion by alespour** [community/community#25262] [Actions]
<p>I have added a configuration matrix with two items to a job that runs on pull request event. I did not change the name of the job (<code>lint-test</code>). Now there are 3 checks in actions, but the last one is not longer in the workflow, it is obsolete and obviously never finishes. How can I get rid of it, please? Thank you very much in advance.</p>
<p><code>helm-charts/test / lint-test (1.21) (pull request) Successful ...</code><br>
<code>helm-charts/test / lint-test (1.22) (pull request) Successful ...</code><br>
<code>lint-test Expected — Waiting for status to be reported</code></p>
<p>PR example: <a href="https://github.com/influxdata/helm-charts/pull/415" rel="noopener nofollow ugc">https://github.com/influxdata/helm-charts/pull/415</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It looks like you (or someone else) have configured that job with the old version as a required check under branch protection rules. If you don’t want the check any more, you’ll have to delete it from the relevant branch protection rule. Adding the updated job(s) at the same time would probably make sense.</p>
